render-config.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+## This script renders configuration files needed for development.
```

Comment:
Is the terminology borrowed from some other framework? Just seems like this is grabbing the SA key, and not doing much in the way of config rendering

Reply:
Yes, this was a copy/paste error. We have a similar script in most of the Enterprise Terra git repos and it's generally called `render-config.sh`. I'm not sure where that name came from, but agreed it's not a great description. Changed the filename to `pull-credentials.sh` and updated the description.